tests/test_remote_operator.py

```diff
@@ -6,6 +6,12 @@
 import pytest
 
 
+@pytest.fixture(scope="module", autouse=False)
```

Comment:
@PProfizi that's one of the ones that are not used, right ?

Reply:
@anslpa Yes, since autouse=False and it does not appear anywhere else.
I can remove them if you prefer, it would be clearer.